src/test/java/com/jcabi/dynamo/AttributeUpdatesTest.java

```diff
@@ -0,0 +1,265 @@
+            Matchers.equalTo(firstvalue)
+        );
+    }
+
+    /**
+     * AttributesUpdates can return its entry Set.
+     */
+    @Test
+    public void returnsEntries() {
+        final AttributeUpdates attr = new AttributeUpdates()
+            .with("key5", "value5").with("key6", "value7");
+        MatcherAssert.assertThat(
+            attr.entrySet().size(),
+            Matchers.is(2)
+        );
+    }
+
+    /**
+     * AttributesUpdates can return an entry by the key.
+     */
+    @Test
+    public void getsEntry() {
+        final String key = "key10";
+        final AttributeUpdates attr = new AttributeUpdates()
+            .with(key, "value10");
+        MatcherAssert.assertThat(
+            attr.get(key),
+            Matchers.notNullValue()
+        );
+    }
+
+    /**
+     * AttributesUpdates can tell if it contains a key.
+     */
+    @Test
+    public void containsKey() {
+        final String key = "key11";
+        final AttributeUpdates attr = new AttributeUpdates()
+            .with(key, "value11").with("otherkey1", "othervalue1");
+        MatcherAssert.assertThat(
+            attr.containsKey(key),
+            Matchers.is(Boolean.TRUE)
+        );
+    }
+
+    /**
+     * AttributesUpdates can tell if it contains a value.
+     */
+    @Test
+    public void containsValue() {
+        final String value = "attrv";
+        final AttributeUpdates attr = new AttributeUpdates()
+            .with("attrkey", value).with("otherkey", "othervalue");
+        MatcherAssert.assertThat(
+            attr.containsValue(
+                new AttributeValueUpdate(
+                    new AttributeValue(value), AttributeAction.PUT
+                )
+            ),
+            Matchers.is(Boolean.TRUE)
+        );
+    }
+
+    /**
+     * AttributesUpdates can return a string representation of itself.
+     */
+    @Test
+    public void canTurnToString() {
+        final AttributeUpdates attr = new AttributeUpdates()
+            .with("onekey", "onevalue").with("secondkey", "secondvalue");
+        MatcherAssert.assertThat(
+            attr.toString(),
+            Matchers.equalTo(
+                StringUtils.join(
+                    "onekey={Value: {S: onevalue,},Action: PUT}; ",
+                    "secondkey={Value: {S: secondvalue,},Action: PUT}"
+                )
+            )
+        );
+    }
+
+    /**
+     * AttributesUpdates can add the content of another Map.
+     */
+    @Test
+    public void addsMap() {
+        final AttributeUpdates attr = new AttributeUpdates();
+        MatcherAssert.assertThat(attr.size(), Matchers.is(0));
+        MatcherAssert.assertThat(
+            attr.with("testkey8", new AttributeUpdates().with("key", "value"))
+                .size(),
+            Matchers.is(1)
+        );
+    }
+
+    /**
+     * AttributesUpdates.put(...) throws exception when called.
+     */
+    @Test(expected = UnsupportedOperationException.class)
```

Comment:
@amihaiemil please see: https://github.com/teamed/qulice/issues/668 - no big harm here, but future version of Qulice would prohibit such code